### ISSUE TITLE
no-output-native should handle new ouput() function

### ISSUE THREAD

**Issue by 5im0n** [angular-eslint/angular-eslint#2004]
```JSON
{
  "rules": {
  "@angular-eslint/no-output-native": [
      "error"
    ]
  }
}
```

✅ Valid Code
```Typescript
readonly click = output<KeyboardEvent>();
```
The code above should be invalid.

❌ Invalid Code
```HTML
 @Output() readonly click = new EventEmitter();
```

**Versions**

| package                                   | version |
| ----------------------------------        | ------- |
| `@angular-eslint/eslint-plugin-template`  | `18.2.0` |
| `@angular-eslint/template-parser`         | `18.2.0` |
| `@typescript-eslint/parser`               | `7.18.0` |
| `ESLint`                                  | `8.57.0` |
| `node`                                    | `20.16.0` |

```sh
@angular-devkit/architect    0.1801.3
@angular-devkit/core         18.1.3
@angular-devkit/schematics   18.1.3
@schematics/angular          18.1.3
rxjs                         7.8.1
typescript                   5.5.4
zone.js                      0.14.10

```

- [X ] I have tried restarting my IDE and the issue persists.
- [ X] I have updated to the latest supported version of the packages and checked my `ng version` output per the instructions given here.

**Comment by JamesHenry** [comment]
Thanks, would you like to submit a PR for this @5im0n?

**Comment by 5im0n** [comment]
@JamesHenry After further consideration, this issue isn't limited to just this rule. It also impacts `@angular-eslint/no-input-rename`, `@angular-eslint/no-output-on-prefix`, and `@angular-eslint/no-output-rename`.

To address this, we need to create new selectors to handle the updated `input()` and `output()` functions.

I could take care of this around mid-September.
